.travis.yml

```diff
@@ -39,4 +39,4 @@ matrix:
     jdk: oraclejdk8
     sudo: required
     before_install:
-    - curl https://raw.githubusercontent.com/scala-native/scala-native/v0.3.6/bin/travis_setup.sh | travis_wait 15 bash -x
+    - curl https://github.com/scala-native/scala-native/blob/master/bin/travis_setup.sh | travis_wait 15 bash -x
```

Comment:
This is going to download html content for that file, not the raw data. Naturally, bash wouldn't be able to run that. Hence, the failure.

Reply:
Correct link: https://raw.githubusercontent.com/scala-native/scala-native/master/bin/travis_setup.sh

Reply:
I've corrected the link, but it seems to fail regardless...
(Thx for you help btw)